client.go

```diff
@@ -51,6 +51,14 @@ type Client interface {
 	Stop() error
 }
 
+// Broadcaster defines a client that can send messages to the registered conversations
+// following certain condition
+type Broadcaster interface {
```

Comment:
I think this should be in the `Client` interface, rather than on its own interface